CDP4Common/AutoGenThingPropertyAccessor/ActionItemPropertyAccessor.cs

```diff
@@ -68,21 +68,23 @@ public override object QueryValue(string path)
             switch (propertyName)
             {
                 case "iid":
-                    return base.QueryThingValues(pd.Input);
```

Comment:
its been a while since i implemented this, but i was very specific when choosing `base` instead of `this`. Can you please revert?